dts/common/yaml/zephyr-i2c.yaml

```diff
@@ -0,0 +1,19 @@
+---
+title: Zephyr specific I2C properties
+id: i2c
+version: 0.1
+
+description: >
+    This binding gives Zephyr specific I2C properties
+
+properties:
+  - zephyr,speed :
```

Comment:
I'm not convinced we need this property. The dts/common/yaml/i2c.yaml file defines "clock-frequency" property which defines maximum bus speed supported by all devices attached to the I2C bus. We can use it to set the default bus speed at boot. I can't think of a use case where we would like to set the bus speed at boot to a lower value.